packages/terra-tabs/src/clinical-lowlight-theme/Tabs.module.scss

```diff
@@ -55,6 +56,11 @@
     --terra-tabs-structural-focus-outline: 2px dashed #c8cacb;
     --terra-tabs-structural-focus-outline-offset: -5px;
 
+    --terra-tabs-addButton-focus-background-color: #222a2e;
+    --terra-tabs-addButton-focus-color: initial;
+    --terra-tabs-addButton-focus-outline: 2px dashed #c8cacb;
+    --terra-tabs-addButton-focus-outline-offset: -5px;
```

Comment:
same here can we have this styles updated with -2px

Reply:
Done https://github.com/cerner/terra-framework/commit/09f89f4169db28da2af3774ea6bd197df5344813